packages/grafana-data/src/types/pluginExtensions.ts

```diff
@@ -167,6 +167,11 @@ export type PluginExtensionEventHelpers<Context extends object = object> = {
   context?: Readonly<Context>;
   // Opens a modal dialog and renders the provided React component inside it
   openModal: (options: PluginExtensionOpenModalOptions) => void;
+  /**
+   * @alpha @internal
+   * Opens the extension sidebar with the registered component.
+   */
+  openExtensionSidebar?: (componentTitle: string, context: Context) => void;
```

Comment:
Maybe just name this `openSidebar`?

Reply:
I think that makes sense. Changed in a90666553aa96ecfa1f055a26de58824730322a4